clang-tools-extra/clang-tidy/kokkos/KokkosMatchers.h

```diff
@@ -42,9 +42,21 @@ AST_MATCHER(CallExpr, isKokkosParallelCall) {
   return false;
 }
 
-bool explicitlyUsingHostExecutionSpace(CallExpr const *CE,
-                                       std::string const &RegexString);
+AST_MATCHER_P(Decl, matchesAttr, std::string, RegExp) {
```

Comment:
I should probably extract this into a function called `matchesAnnotation` and then call that here, mainly because the function can be reused for matching the host/device type annotations as well.